test.js

```diff
@@ -1,19 +1,51 @@
 import test from 'ava';
 import filterObject from './index.js';
 
-test('function predicate', t => {
+test('function predicate returns a boolean', t => {
 	t.is(Object.keys(filterObject({foo: true, bar: false}, () => true)).length, 2);
 	t.is(Object.keys(filterObject({foo: true, bar: false}, () => false)).length, 0);
-	t.is(Object.keys(filterObject({foo: true, bar: false}, (key, value) => value === true)).length, 1);
+});
+
+test('function predicate passes the key as argument', t => {
+	t.true(filterObject({foo: true}, key => key === 'foo').foo);
+});
+
+test('function predicate passes the value as argument', t => {
+	t.is(filterObject({foo: 'test'}, (key, value) => value === 'test').foo, 'test');
+});
+
+test('function predicate passes the object as argument', t => {
+	t.true(filterObject({foo: true}, (key, value, object) => object.foo).foo);
 });
 
 test('array predicate', t => {
-	const filteredKeys = Object.keys(filterObject({foo: true, bar: false}, ['foo']));
-	t.is(filteredKeys[0], 'foo');
-	t.is(filteredKeys.length, 1);
+	t.deepEqual(Object.keys(filterObject({foo: true, bar: false}, ['foo'])), ['foo']);
+});
+
+test('symbol properties', t => {
+	const symbol = Symbol('test');
+	const input = {[symbol]: true};
+	t.is(filterObject(input, key => key === symbol)[symbol], undefined);
```

Comment:
I believe this is a bug? `filter-obj` should not omit symbol properties.
This is currently doing so because `Object.entries()` does not return symbol properties, unlike `Object.getOwnPropertySymbols()` or `for (const key in object)`.